fullykiosk/__init__.py

```diff
@@ -1,95 +1,123 @@
+import aiohttp
 import json
 import logging
 import requests
```

Comment:
leftover import?

Reply:
It's not 100% leftover as I reference it on line 95, but the strict type definition is not completely necessary of course. I can remove it if you think that's cleaner.

Reply:
Doh :D fixed...